src/storage/models.rs

```diff
@@ -1,15 +1,15 @@
 use std::iter;
 
 use rand::Rng;
-use serde::Serialize;
-
-use crate::storage::DateTime;
+use serde::{ser::SerializeStruct, Serialize, Serializer};
 
 const DEFAULT_LIMIT_SIZE: usize = 20;
 const DEFAULT_SLUG_LENGTH: usize = 8;
 
+type DateTime = chrono::DateTime<chrono::Utc>;
```

Comment:
Where do we use this?

Reply:
Oh, I see how this can be confusing: for this type alias I've decided to reuse the same name that I previously used for our struct-wrapper around `chrono:DateTime`. Keeping the name allowed me to not change the lines L29 and L32 below.